hash/zx81_cass.xml

```diff
@@ -1352,8 +1394,28 @@ Known dumps not yet added (as of 2023-06-14):
 		</part>
 	</software>
 
-	<software name="chessty" cloneof="chess">
-		<description>Chess (Typed)</description>
+	<software name="chess_d" cloneof="chess">
```

Comment:
Please give it it's actual name 'xadrez'.

Reply:
Yeah, even if you were going to go for “parent plus suffix”, something mnemonic like `chess_pt` would be a lot more descriptive than a non-descript letter suffix.